modules/arm_plugin/src/arm_converter/arm_converter.hpp

```diff
@@ -63,12 +63,17 @@ struct Argument<arm_compute::ITensor*> {
     arm_compute::Tensor     _notPaddedTensor;
 };
 
+struct HostTensors {
+    ngraph::HostTensorVector _hosts;
+    const ngraph::Node*      _node = nullptr;
```

Comment:
Why do you need to have node raw pointer here?

Reply:
it is used to map host tensors to backend tensors that are stored in a map indexed by node friendly name.

Reply:
If i can extract this node from `ngraph::tensors` i can remove this structure

Reply:
It will be better to extend ngraph::runtime::Tensor API. But in any case we cannot guarantee that HostTensor always has the name.

Do you really need to have the tensor name? Can you guarantee that tensor name exists?

Reply:
This PR removes usage of tensor name in ARM Plugin

Reply:
Yes I understood. But at the same moment [here](https://github.com/openvinotoolkit/openvino_contrib/pull/134/files#diff-c711f021618ec3aa70ee7aa1003d3d952aad1b72208f1b4c99ca94e68e1f3e3cR400) you use the friendly name to connect node with tensor. Why do you need it?

Reply:
What if operation has several output tensors? Do these tensors have one name?

Reply:
If tensor names are not used in `ngraph` reference computation code it dose not mater. We map backend tensors to elements of  `ngraph::HostTensorVector` serially one by one.

Reply:
`ngraph::Tensor` name was used just to find what `ngraph::Node` produce the tensor. This PR just store the pointer to `ngraph::Node` near the tensor so we drop usage of deprecated `get_name` API

Reply:
@apankratovantonp I really dislike such extension, because it looks like a WA. Because:

- HostTensor (if we are talking about get_name()) can be created without the name
- Tensor name can be different and it is not equal to node name
- We have name collisions if Node has several outputs

Maybe can we discuss it offline?

Reply:
Discussed offline.